resources/public/api/conf/P1.0/examples/contact-details-response.json

```diff
@@ -0,0 +1,21 @@
+{
+  "_links": {
+    "self": {
+      "href": "/individuals/details/contact-details?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430"
+    }
+  },
+  "contactDetails": [
```

Comment:
Are we happy with this format? 

If we are building domain models for OGDs then we can replay this however we like? 

Suggest 

```
"contactDetails": [
    {
       "mobileTelephone": "07123 987654"
     },
     {
       "daytimeTelephone": "07123 987654"
     },
     {
       "eveningTelephone": "07123 987654"
     }
   ]
```

The schema would have to change also. Addresses are fairly standard and look OK.

Reply:
In all honesty, I think maybe collapsing to a single object with mobile, daytime and evening would be better. E.g.
`"contactDetails: {
          "mobileTelephone" : "07123 987654"
          "daytimeTelephone" : "07123 987654"
          "eveningTelephone" : "07123 987654"
}"`
If we're making better mappings may as well but this would require further mapping. Happy to discuss further

Reply:
Sure. Mine was simply an example. Feel free to go with the above. :-)